Source/DataSources/CzmlDataSource.js

```diff
@@ -600,7 +600,10 @@ define([
         var hasInterval = defined(combinedInterval) && !combinedInterval.equals(Iso8601.MAXIMUM_INTERVAL);
 
         if (!isReference) {
-            referenceFrame = defaultValue(ReferenceFrame[packetData.referenceFrame], undefined);
+            if (defined(packetData.referenceFrame)) {
+                referenceFrame = ReferenceFrame[packetData.referenceFrame];
+            }
+            referenceFrame = defaultValue(referenceFrame, ReferenceFrame.FIXED);
```

Comment:
> For example, if CZML can't find or fails to load a glTF, currently we throw and break the render loop

This has nothing to do with CZML and is a `Model` primitive behavior.  It also changed in 1.17 and no longer throws (which was a bug).